src/components/RoomView/RoomView.jsx

```diff
@@ -1,10 +1,8 @@
-/* eslint-disable no-unused-vars */
```

Comment:
lots of console logs on this file, I suppose some of them might be useful for future use so maybe just clean up the ones you think are unnecessary now?